tests/sentinel/tests/14-config-set.tcl

```diff
@@ -0,0 +1,36 @@
+source "../tests/includes/init-tests.tcl"
+
+test "SENTINEL CONFIG SET handles multiple variables" {
+    foreach_sentinel_id id {
+        assert_equal {OK} [S $id SENTINEL CONFIG SET resolve-hostnames yes announce-port 1234]
+    }
+
+    after 2000
```

Comment:
Why we need to wait? The change of both params is immediate

Reply:
Hello Good Morning,
The wait of 2 seconds was added as a precaution as in the past some components of the daily CI failed in certain machines. There is also a similar wait added in
https://github.com/redis/redis/blob/11b071a22bd1662166213e86f7bb7394a945ea00/tests/sentinel/tests/09-acl-support.tcl#L43
If you think this is not needed then I can remove it. Thanks!

Reply:
good morning Wen :) 

Better not write redundant sleeps because:  
- It impact our execution time.
- Will confuse somebody else that will read the code.
- And serve as bad reference  to other tests to follow.

Reply:
Sure that makes sense. I have removed the wait from the test. Thanks for the explanation